apps/expo/App.tsx

```diff
@@ -22,6 +23,16 @@ Sentry.init({
   enableInExpoDevelopment: false,
 });
 
+const scheme = `io.showtime${
+  process.env.NODE_ENV === "development" ? ".development" : ""
+}`;
+
+configureWalletMobileSDK({
+  callbackURL: new URL(`${scheme}://wsegue`), // TODO: Use universal link to prevent warning message in CB Wallet
```

Comment:
What do you mean by "universal link"?

Reply:
[Universal Links](https://developer.apple.com/documentation/xcode/allowing-apps-and-websites-to-link-to-your-content) would allow you to deeplink into the Showtime app with a url like `https://showtime.xyz/wsegue`
Seems like it is already set up here: https://showtime.xyz/.well-known/apple-app-site-association 
Maybe a path needs to be added for `/wsegue` in there.

Reply:
Ah yes! OK thanks, I'll handle this